core/src/serve_repair.rs

```diff
@@ -269,21 +269,33 @@ impl ServeRepair {
         Ok(out)
     }
 
-    pub fn repair_request(&self, repair_request: &RepairType) -> Result<(SocketAddr, Vec<u8>)> {
+    pub fn repair_request(
```

Comment:
@carllin @sagar-solana wdyt?  this does a single cluster info lookup per repair request.